src/demux.js

```diff
@@ -277,6 +275,18 @@ class DemuxSink extends Writable {
 
         const { statsSessionId, type, data } = request;
 
+        // save the last sequence number to notify the frontend
+        if (Array.isArray(data)) {
```

Comment:
`[\"conferenceStartTimestamp\",null,1669298796000,1669298796327]`
vs
`identity,,[object Object],1669298796970 ]`

Reply:
but it's still an array isn't it? We don't deserialize the [object Object] at this point.

Reply:
Or actually it's the other way around, we always have to JSON.parse because data will always be an encoded string at this point, Array.isArray should never return true here.

Reply:
I understand the case now, dismiss my previous comment.